static/css/friend.css

```diff
@@ -0,0 +1,130 @@
+#article-container {
+	word-wrap: break-word;
+	overflow-wrap: break-word;
+}
+
+#article-container a {
+	color: #49b1f5;
+}
+
+#article-container a:hover {
+	text-decoration: underline;
+}
+
+#article-container img {
+	margin: 0 auto 0.8rem;
+}
+
+.flink#article-container .friend-list-div > .friend-div a .friend-info,
+.flink#article-container .friend-list-div > .friend-div a .friend-name {
```

Comment:
This seems to assume that some container with `class="flink" id="article-container"` exists, but the website's template & the shortcode itself does not generate such a container. `.friend-list-div` also has the same issue. This design would make it hard for the users to use this shortcode correctly. We should create another shortcode that generates the container, which is similar to the nested shortcode here: https://gohugo.io/templates/shortcode-templates/#nested-shortcode-image-gallery.